### ISSUE TITLE
After installation, the plugin breaks the build

### ISSUE THREAD

**Issue by vaban-ru** [BazuMax/capacitor-plugin-vk-auth#3]
Hello! Lately, after installing the plugin, I get a build error.
What am I doing:
`yarn add capacitor-plugin-vk-auth`
`npx cap sync`
`yarn capacitor:build`

Get error:
```
> Task :app:processReleaseResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processReleaseResources'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
   > Android resource linking failed
     C:\Users\vtbor\.gradle\caches\transforms-2\files-2.1\5c8e0749fc6c341721d8866e699ff05c\core-1.7.0\res\values\values.xml:105:5-114:25: AAPT: error: resource android:attr/lStar not found.
```

If I remove the plugin, then the build runs fine

**Comment by bazuka5801** [comment]
Hello, please send output of 'npx cap doctor'.

**Comment by vaban-ru** [comment]
> npx cap doctor

```
 Capacitor Doctor

Latest Dependencies:

  @capacitor/cli: 3.3.1
  @capacitor/core: 3.3.1
  @capacitor/android: 3.3.1
  @capacitor/ios: 3.3.1

Installed Dependencies:

  @capacitor/ios: not installed
  @capacitor/cli: 3.1.1
  @capacitor/android: 3.1.1
  @capacitor/core: 3.1.1

[success] Android looking great! 👌
  
  ```

**Comment by vaban-ru** [comment]
Hello @bazuka5801, any updates on this?

**Comment by bazuka5801** [comment]
@webzlodimir Try after installing plugin, click on Android Studio menu item `File -> Invalidate Cache & Restart`

**Comment by vaban-ru** [comment]
@bazuka5801
No it didn't help
![image](https://user-images.githubusercontent.com/31979423/143381021-40c5334f-a252-49ea-b088-1779f6af7f53.png)
![image](https://user-images.githubusercontent.com/31979423/143381045-b96e1de4-0484-4d0b-874e-473c3ad8a240.png)

**Comment by vaban-ru** [comment]
😥😥😥

**Comment by bazuka5801** [comment]
@vaban-ru Fixed in new `3.0.1` version
